apps/language_server/test/providers/execute_command/manipulate_pipes_test.exs

```diff
@@ -760,4 +889,42 @@ defmodule ElixirLS.LanguageServer.Providers.ExecuteCommand.ManipulatePipesTest d
              ) == edited_text
     end
   end
+
+  defp assert_never_raises(text, uri, command) do
```

Comment:
@axelson I've added this helper assertion that travels through the whole text calling the corresponding command ("toPipe" or "fromPipe"). This should ensure that no position in each test file raises in the code. Some bugs have been found and fixed this way already :)